modules/admin/src/main/resources/webapp/messages/messages.properties

```diff
@@ -104,8 +104,8 @@ admin.queue.enqueue.count=Enqueue Count
 admin.queue.dequeue.count=Dequeue Count
 admin.queue.expired.count=Expiry Count
 admin.queue.consumer.count=Consumer Count
-admin.queue.error=Could not retrieve queue statistics.Please inform admin.
-admin.queue.message.error=Could not retrieve message information from the queue.Please inform admin.
+admin.queue.error=Could not retrieve queue statistics. Check http://docs.motechproject.org/en/latest/faq.html
```

Comment:
Can we change this URL to point to an anchor for the specific FAQ questions (rather than the top-level FAQ page)?